validation.py

```diff
@@ -4,18 +4,15 @@
 from games.game_factory import GameFactory
 from models_db import League
 
-# List of allowed modules and their allowed sub-modules
```

Comment:
Why get rid of this?

We need to white list not black list. There is an unlimited number of potentially dangerous modules given that anyone can create one. We need to have a whitelisting solution. Namely we need to list what is allowed and forbit everything else in terms of imports for the games

Reply:
Yep we need whitelist solution. Perhaps something that generates a game list from browsing the filesystem, the same thing that generates a game list on the front end